modules/ixBidAdapter.js

```diff
@@ -52,33 +59,59 @@ function bidToBannerImp(bid) {
 function bidToVideoImp(bid) {
   const imp = bidToImp(bid);
   const videoAdUnitRef = utils.deepAccess(bid, 'mediaTypes.video');
-  const context = utils.deepAccess(bid, 'mediaTypes.video.context');
+  const videoParamRef = utils.deepAccess(bid, 'params.video');
+
   const videoAdUnitAllowlist = [
     'mimes', 'minduration', 'maxduration', 'protocols', 'protocol',
     'startdelay', 'placement', 'linearity', 'skip', 'skipmin',
     'skipafter', 'sequence', 'battr', 'maxextended', 'minbitrate',
     'maxbitrate', 'boxingallowed', 'playbackmethod', 'playbackend',
-    'delivery', 'pos', 'companionad', 'api', 'companiontype', 'ext'
+    'delivery', 'pos', 'companionad', 'api', 'companiontype', 'ext',
+    'playerSize'
```

Comment:
What happens if someone sets h and w on the ad unit?

Reply:
Pbs adapterbtaking approach of preferring h/w if both specified; 1833 in pbs

Reply:
nice, will check it out